plugins/leemons-plugin-academic-calendar/frontend/src/pages/private/regional/index.js

```diff
@@ -106,65 +105,53 @@ export default function RegionalCalendars() {
   const [t] = useTranslateLoader(prefixPN('regionalList'));
   const [, , , getErrorMessage] = useRequestErrorMessage();
   const scrollRef = React.useRef(null);
-  const { classes, cx } = useStyle();
+  const { classes } = useStyle();
   const [isDrawerOpen, setIsDrawerOpen] = useState(false);
   const [userCenters, setUserCenters] = useState();
   const [selectedCenter, setSelectedCenter] = useState();
+  const [center, setCenter] = useState(null);
+  const [selectedConfig, setSelectedConfig] = useState({});
+  const [saving, setSaving] = useState(false);
   const deploymentConfig = useDeploymentConfig({
     pluginName: 'academic-calendar',
     ignoreVersion: true,
   });
-  const [store, render] = useStore({
-    center: null,
-    scroll: 0,
-  });
 
-  const { data: regionalConfigs, refetch: refetchConfigs } = useListRegionalConfigs(
-    store?.center?.id,
-    { enabled: !!store?.center?.id }
-  );
+  const { data: regionalConfigs } = useListRegionalConfigs(center?.id, {
+    enabled: !!center?.id,
+  });
 
-  async function loadRegionalConfigs() {
-    refetchConfigs();
-    store.regionalConfigs = regionalConfigs?.regionalConfigs;
-    render();
-  }
+  const { mutate: deleteRegionalConfigs } = useDeleteRegionalConfigs(center?.id);
+  const { mutate: saveRegionalConfig } = useSaveRegionalConfig(center?.id);
 
-  async function handleOnDeleteRegionalCalendar(id) {
-    try {
-      await deleteRegionalConfig(id);
-      await loadRegionalConfigs();
-      render();
-    } catch (error) {
-      console.error('Error deleting regional config:', error);
-      addErrorAlert(getErrorMessage(error));
-    }
+  function handleOnDeleteRegionalCalendar(configId) {
+    deleteRegionalConfigs(configId, center?.id, {
+      onSuccess: () => {
+        addSuccessAlert('Config deleted successfully');
```

Comment:
Copy not localized

Reply:
localization added, and create a new localization for error alert.